.github/automation/performance/benchdnn_comparison.py

```diff
@@ -76,10 +76,12 @@ def compare_two_benchdnn(file1, file2, check=False):
         r2_ctime[key].append(float(ctime))
 
     exec_failures, ctime_failures = [], []
-    if not check:
-        print(
-            "primitive,exec_base,exec_new,ctime_base,ctime_new,exec_diff,ctime_diff"
-        )
+    if out_file is not None:
+        with open(out_file, "w") as f:
+            f.write(
+                "| problem | base time(ms) | new time(ms) | speedup |\n"
```

Comment:
Can we have something like `(>1 means faster)`.

Also, looking at the wiki, it's hard to understand what base and new are. Please can we have some way of labelling these? Either from the page title or the column headers?